packages/node_modules/@node-red/editor-client/src/sass/mixins.scss

```diff
@@ -15,18 +15,10 @@
  **/
 
 @mixin disable-selection {
-    -webkit-user-select: none;
```

Comment:
I wouldn't remove WebKit because Safari and Firefox browsers support it. Additionally Safari doesn't like `user-select` at all.